### PR TITLE
Java Docs cleanup

### PR DESCRIPTION
**Pull request by Beacze**

### CODE REVIEW

File: Kitodo/src/main/java/org/goobi/production/export/ExportXmlLog.java [hunk 1]
```diff
@@ -95,7 +95,6 @@ public void startExport(Process p, File dest) throws IOException {
      *            the process to export
      * @param os
      *            the OutputStream to write the contents to
-     * @throws IOException
```
Comment (henning-gerhardt) [inline Kitodo/src/main/java/org/goobi/production/export/ExportXmlLog.java]: Why did you remove this entry as this exception is still thrown.

Comment (Beacze) [inline Kitodo/src/main/java/org/goobi/production/export/ExportXmlLog.java]: restored
